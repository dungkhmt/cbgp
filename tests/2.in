50 50
50 125
10 5
14 5
31 7
5 12
18 13
11 44
1 18
31 43
43 5
46 14
21 13
1 8
16 12
20 50
43 1
14 36
27 4
2 1
24 48
12 12
11 4
15 8
13 29
4 1
32 33
43 20
40 41
12 26
34 10
45 30
19 19
11 50
4 35
8 18
3 20
27 6
6 32
16 17
4 6
5 7
7 14
48 44
40 14
13 13
1 12
5 3
2 14
2 8
1 30
44 43
31 39
30 18
8 23
9 15
40 46
43 49
1 13
15 4
12 17
14 33
5 20
9 19
30 30
15 23
31 47
7 7
49 4
25 50
7 39
24 6
1 19
5 1
9 42
17 26
47 43
2 30
6 8
17 17
2 12
12 36
2 36
6 2
41 7
19 18
44 33
9 8
10 3
32 2
18 42
43 7
1 3
43 50
10 20
5 34
9 4
7 28
40 7
37 17
15 27
43 4
20 25
24 38
46 7
19 22
1 20
9 18
