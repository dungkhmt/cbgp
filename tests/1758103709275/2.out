(29, 15), (19, 0), (5, 7), (23, 12), (31, 43), (25, 26), (43, 27), (19, 2), (8, 10), (8, 12), (24, 1), (9, 22), (36, 12), (0, 28), (10, 11), (17, 34), (9, 10), (30, 25), (39, 35), (10, 20), (40, 0), (40, 44), (23, 26), (9, 46), (2, 4), (29, 38), (48, 8), (0, 1), (2, 45), (29, 3), (32, 48), (2, 8), (2, 26), (4, 35), (17, 49), (9, 43), (19, 8), (5, 3), (4, 39), (10, 18), (0, 5), (4, 14), (8, 32), (21, 1), (13, 4), (49, 10), (29, 40), (25, 45), (12, 26), (12, 30), (3, 0), (45, 26), (20, 28), (17, 20), (39, 42), (46, 1), (1, 10), (17, 0), (22, 2), (20, 49), (24, 9), (22, 39), (48, 18), (47, 10), (36, 25), (36, 26), (35, 9), (23, 8), (9, 2), (1, 17), (11, 3), (10, 48), (40, 7), (29, 33), (17, 10), (2, 6), (3, 44), (32, 18), (10, 29), (38, 44), (15, 0), (10, 3), (5, 44), (38, 3), (0, 10), (41, 37), (13, 9), (9, 6), (23, 2), (34, 28), (14, 2), (34, 20), (27, 9), (8, 0), (27, 6), (15, 37), (33, 38), (0, 20), (2, 39), (8, 18), (15, 16), (27, 31), (24, 46), (25, 12), (12, 27), (9, 12), (0, 2), (28, 17), (40, 38), (1, 9), (30, 36), (3, 8), (22, 35), (42, 4), (10, 12), (42, 14), (46, 13), (18, 12), (16, 37), (47, 0), (40, 5), (15, 47), (10, 15), 
numberIntersection=0 
minAngle=0.003553839603861242 
sumAngle=31.47639472588613 
minEdgeLength=4.0 
sumEdgeLength=5615.494696424898 
minDistance=0.4681645887845223 
sumDistance=368772.96746720077 

0.0000000000 0.0035538396 31.4763947259 4.0000000000 5615.4946964249 0.4681645888 368772.9674672008
0: (92, 184),
1: (70, 88),
2: (120, 68),
3: (148, 20),
4: (100, 108),
5: (158, 36),
6: (86, 72),
7: (162, 40),
8: (146, 24),
9: (82, 84),
10: (12, 12),
11: (148, 16),
12: (62, 40),
13: (98, 112),
14: (104, 96),
15: (2, 4),
16: (0, 0),
17: (68, 92),
18: (62, 36),
19: (122, 72),
20: (56, 100),
21: (76, 92),
22: (96, 88),
23: (128, 44),
24: (76, 96),
25: (116, 60),
26: (126, 48),
27: (78, 72),
28: (60, 108),
29: (180, 8),
30: (116, 56),
31: (84, 76),
32: (62, 32),
33: (168, 24),
34: (60, 104),
35: (94, 96),
36: (118, 52),
37: (188, 0),
38: (168, 28),
39: (98, 92),
40: (162, 44),
41: (184, 8),
42: (100, 96),
43: (82, 80),
44: (166, 32),
45: (118, 64),
46: (90, 128),
47: (10, 16),
48: (60, 28),
49: (56, 96),
92 184 70 88 120 68 148 20 100 108 158 36 86 72 162 40 146 24 82 84 12 12 148 16 62 40 98 112 104 96 2 4 0 0 68 92 62 36 122 72 56 100 76 92 96 88 128 44 76 96 116 60 126 48 78 72 60 108 180 8 116 56 84 76 62 32 168 24 60 104 94 96 118 52 188 0 168 28 98 92 162 44 184 8 100 96 82 80 166 32 118 64 90 128 10 16 60 28 56 96 
0: (92, 184), 1: (70, 88), 2: (120, 68), 3: (148, 20), 4: (100, 108), 5: (158, 36), 6: (86, 72), 7: (162, 40), 8: (146, 24), 9: (82, 84), 10: (12, 12), 11: (148, 16), 12: (62, 40), 13: (98, 112), 14: (104, 96), 15: (2, 4), 16: (0, 0), 17: (68, 92), 18: (62, 36), 19: (122, 72), 20: (56, 100), 21: (76, 92), 22: (96, 88), 23: (128, 44), 24: (76, 96), 25: (116, 60), 26: (126, 48), 27: (78, 72), 28: (60, 108), 29: (180, 8), 30: (116, 56), 31: (84, 76), 32: (62, 32), 33: (168, 24), 34: (60, 104), 35: (94, 96), 36: (118, 52), 37: (188, 0), 38: (168, 28), 39: (98, 92), 40: (162, 44), 41: (184, 8), 42: (100, 96), 43: (82, 80), 44: (166, 32), 45: (118, 64), 46: (90, 128), 47: (10, 16), 48: (60, 28), 49: (56, 96), 
numberIntersection=0 minAngle=0.003553839603861242 sumAngle=31.47639472588613 minEdgeLength=4.0 sumEdgeLength=5615.494696424898 minDistance=0.4681645887845223 sumDistance=368772.96746720077 
Time taken: 1518.695 s
